source/_patterns/01-atoms/icons/icon-video.twig

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30"><title>Video Icon</title><path d="M5.71,4.34A5.62,5.62,0,0,0,0,9.84V20.16a5.62,5.62,0,0,0,5.71,5.5H24.29A5.62,5.62,0,0,0,30,20.16V9.84a5.62,5.62,0,0,0-5.71-5.5Zm0,1.38H24.29a4.18,4.18,0,0,1,4.28,4.12V20.16a4.18,4.18,0,0,1-4.28,4.12H5.71a4.18,4.18,0,0,1-4.28-4.12V9.84A4.18,4.18,0,0,1,5.71,5.72Zm5.72,4.81v8.94a.71.71,0,0,0,1.08.6l7.86-4.47a.67.67,0,0,0,.36-.59.69.69,0,0,0-.36-.6L12.51,9.94a.77.77,0,0,0-.42-.1A.7.7,0,0,0,11.43,10.53Zm1.43,1.22L18.58,15l-5.72,3.27V11.75Z" fill-rule="evenodd"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30"><title>Video Icon</title><path d="M20.49,13.63,13.14,9.44a1.6,1.6,0,0,0-.92-.21,1.62,1.62,0,0,0-1.51,1.58v8.38A1.58,1.58,0,0,0,12,20.76l.29,0a1.59,1.59,0,0,0,.8-.21l7.35-4.18A1.6,1.6,0,0,0,21.31,15,1.59,1.59,0,0,0,20.49,13.63Zm-6.56,2.82V13.56L16.46,15Z"/><path d="M23.71,4.07H6.29A6.21,6.21,0,0,0,0,10.17v9.66a6.21,6.21,0,0,0,6.29,6.1H23.71A6.21,6.21,0,0,0,30,19.83V10.17A6.21,6.21,0,0,0,23.71,4.07Zm3.08,6.1v9.66a3,3,0,0,1-3.08,2.93H6.29a3,3,0,0,1-3.08-2.93V10.17A3,3,0,0,1,6.29,7.24H23.71A3,3,0,0,1,26.79,10.17Z"/></svg>
```

Comment:
should this have a `o-video-icon` class?

Reply:
Nah, I usually don't add the class names to the `<svg>` elements. I add that to the wrapping element.